junit-platform-engine/src/main/java/org/junit/platform/engine/support/descriptor/MethodSource.java

```diff
@@ -167,6 +175,67 @@ public boolean equals(Object o) {
 				&& Objects.equals(this.methodParameterTypes, that.methodParameterTypes);
 	}
 
+	/**
+	 * Get the {@linkplain Class Java class} of this source.
+	 *
+	 * <p>If the {@link Class} was not provided, but only the name, this method
+	 * attempts to lazily load the {@link Class} based on its name and throws a
+	 * {@link PreconditionViolationException} if the class cannot be loaded.
+	 *
+	 * @since 5.7
+	 * @see #getClassName()
+	 */
+	public final Class<?> getJavaClass() {
```

Comment:
Please add missing `@API` declaration.

Reply:
ok. I assume STABLE is the correct value

Reply:
Yes, STABLE should be fine.